features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mysql.sql

```diff
@@ -589,3 +589,10 @@ DM_DEVICE.ID = DM_DEVICE_DETAIL.DEVICE_ID
 ORDER BY TENANT_ID, DEVICE_ID;
 
 -- END OF DASHBOARD RELATED VIEWS --
+
+-- TEMP TABLE REQUIRED FOR DATA ARCHIVAL JOB
+CREATE TABLE IF NOT EXISTS DM_ARCHIVED_OPERATIONS (
```

Comment:
Why only this table is included in the cdm, and the rest of the archival tables are in the archival folder?

Reply:
@sinthuja This is the temp table that is used when moving records to archival tables.